pkg/api/common/versions.go

```diff
@@ -109,6 +109,7 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.14.0-alpha.1": false,
 	"1.14.0-alpha.2": true,
```

Comment:
remove alpha.2?

Reply:
I thought in the past we left all alpha + beta versions enabled, then set them all to `false` when there was an official release to replace them. But I see we disabled alpha.1 so I assume we're applying our support policy to prereleases as well? Works for me.

Reply:
yeah I'm not sure, I saw that alpha.1 was deprecated which is why I asked. I don't think we're used to having more than 1 pre-release. Either way is fine I think, it doesn't matter much.